src/pages/index.js

```diff
@@ -59,57 +59,119 @@ const cardSection = new Section(
   selectors.cardSection
 );
 
+// Popup instances
 const imagePopup = new PopupWithImage(selectors.previewImagePopup);
+const profileEditPopup = new PopupWithForm(
+  selectors.profileEditPopup,
+  (formData) => {
+    return api
+      .updateUserInfo({
+        name: formData.title,
+        about: formData.description,
+      })
+      .then((updatedData) => {
+        userInfo.setUserInfo({
+          name: updatedData.name,
+          job: updatedData.about,
+        });
+        profileEditPopup.close();
+      })
+      .catch((err) => console.error("Error updating profile:", err));
```

Comment:
`catch` is not needed in `index.js` if you have it inside `PopupWithForm`. It should be only at the very end of the request

**Please, correct it everywhere in the code**

Reply:
Ok. Thank you.